lib/app_localizations.dart

```diff
@@ -677,6 +677,84 @@ class AppLocalizations {
       desc: 'Body of the dialog telling the user they have lost their internet connection',
     );
   }
+
+  // Choose roles
+
+  String get chooseGameRoles {
+    return Intl.message(
+      'Choose the game roles',
+      name: 'chooseGameRoles',
+      desc: 'The room owner has to pick the roles that will be in the game',
+    );
+  }
+
+  String get someoneElseChoosesGameRoles {
+    return Intl.message(
+      'The person who created the room is choosing the game roles',
```

Comment:
I've done this in a separate commit. An error flashes before you see the right screen and I don't know why. It says it can't find the owner but it displays it a moment later.

Reply:
Probably because the app hasn't loaded the players from firestore yet.